src/mlpack/methods/ann/init_rules/glorot_init.hpp

```diff
@@ -89,8 +91,9 @@ class GlorotInitializationType
    * @param cols Number of columns.
    * @param slices Number of slices.
    */
-  template<typename eT>
-  void Initialize(arma::Cube<eT>& W,
+  template<typename CubeType,
+      typename std::enable_if_t<IsCube<CubeType>::value, bool> = false>
```

Comment:
For the same reason, the `enable_if` also isn't needed here, since the three `size_t` arguments differentiate this from the `Mat` case.